configuration/external_parameters.rst

```diff
@@ -124,7 +124,7 @@ You can now reference these parameters wherever you need them.
 
         </container>
 
-    .. code-block:: php
+    ::
```

Comment:
This change is wrong as this is part of a `configuration-block` block. Reverted in #9368.

Reply:
Woops, sorry for this!